### ISSUE TITLE
Weird type-casting syntax error in the setHighTempC and setLowTempC

### ISSUE THREAD

**Issue by alporak** [sparkfun/SparkFun_TMP102_Arduino_Library#5]
```
 // Split temperature into separate bytes
  if(extendedMode)	// 13-bit mode
  {
	registerByte[0] = int(temperature)>>5;
    registerByte[1] = (int(temperature)<<3);
  }
  else	// 12-bit mode
  {
	registerByte[0] = int(temperature)>>4;
    registerByte[1] = int(temperature)<<4;
  }
```
 can't be compiled in non-arduino platforms due to syntax errors. Code compiles and runs on arduino.

**Comment by alporak** [comment]
issue can be fixed by typing it as follows:
```
// Split temperature into separate bytes
	if (extendedMode)	// 13-bit mode
	{
		registerByte[0] = ((int)(temperature)) >> 5;
		registerByte[1] = ((int)(temperature)) << 3;
	} else	// 12-bit mode
	{
		registerByte[0] = ((int)(temperature))>> 4;
		registerByte[1] = ((int)(temperature))<< 4;
	}
```

**Comment by nseidle** [comment]
Thanks for reporting! Agreed. Should be fixed in v1.1.2.
